.github/workflows/build.yml

```diff
@@ -29,7 +29,25 @@ jobs:
       fail-fast: false
     steps:
       - uses: actions/checkout@v2
+      - name: Cache Bottlerocket tools build artifacts
```

Comment:
```suggestion
      # This speeds up the publish-test script, which would otherwise need to
      # build tuftool and pubsys every time, saving several minutes.  We don't
      # cache CARGO_HOME because that's shared with the OS build.
      - name: Cache Bottlerocket tools build artifacts
```